AUTHORS

```diff
@@ -98,6 +98,7 @@ Also thanks to:
     * Max621
     * Max Ugrumov (katzsmile)
     * Michael Rätzel
+		* Michael Silber (frühstück)
```

Comment:
Please use spaces not tabs in this file.

Reply:
Sorry, will do better next time. That came from Tabs only in C# : )

Should i change it?